Formula/djl-bench.rb

```diff
@@ -0,0 +1,38 @@
+class DjlBench < Formula
+  desc "Command-line tool that helps to benchmark the model on different platforms"
+  homepage "https://github.com/deepjavalibrary/djl/tree/master/extensions/benchmark"
+  url "https://publish.djl.ai/djl-bench/0.12.0/benchmark-0.12.0.tar"
+  sha256 "778df005951c546c4e8e4e22dd62e1bc64b413334e0951ffbd8d06be5cc6ce19"
+  license "Apache-2.0"
+
+  depends_on "openjdk"
+
+  def install
+    rm_rf Dir["bin/*.bat"]
+    prefix.install_metafiles
+    mv "bin/benchmark", "bin/djl-bench"
+    libexec.install Dir["*"]
+    djl_env = { APP_HOME: "${APP_HOME:-#{var}}" }
+    djl_env.merge!(Language::Java.overridable_java_home_env)
+    (bin/"djl-bench").write_env_script "#{libexec}/bin/djl-bench", djl_env
+    exec "git clone --recursive https://github.com/apache/incubator-mxnet mxnet"
+    sleep 30
+    exec "cd mxnet"
+    exec "sudo apt-get update"
+    sleep 30
+    exec "sudo apt-get install -y build-essential git ninja-build ccache libopenblas-dev libopencv-dev cmake"
```

Comment:
The macOS runners have no `apt-get`. `git` is already installed, so there's no need to install it. Let's skip the use of `ccache`. For the others, you can do
```ruby
depends_on "cmake"
depends_on "ninja"
depends_on "openblas"
depends_on "opencv"
```
You should add `=> :build` where appropriate (presumably this applies to `ninja` and `cmake`).

Reply:
It's probably also simpler to make `mxnet` a separate formula so this one can just declare that as a dependency.